src/lib/yang/data.lua

```diff
@@ -1012,6 +1013,270 @@ function xpath_printer_from_grammar(production, print_default, root)
 end
 xpath_printer_from_grammar = util.memoize(xpath_printer_from_grammar)
 
+function influxdb_printer_from_grammar(production, print_default, root)
+   if not root then root = '' end
+   if root and #root == 1 and root:sub(1, 1) == '/' then
+      root = ''
+   end
+   local handlers = {}
+   local translators = {}
+   local function printer(keyword, production, printers)
+      return assert(handlers[production.type])(keyword, production, printers)
+   end
+   local function escape_double_quotes (value)
+      assert(type(value) == 'string')
+      return value:gsub('"', '\\"')
+   end
+   local function escape_value (primitive_type, value)
+      if primitive_type == 'decimal64' then
+         return ("%.2f"):format(value)
+      elseif primitive_type == 'string' then
+         return '"'..escape_double_quotes(value)..'"'
+      elseif primitive_type == 'boolean' then
+         return value and 'true' or 'false'
+      else
```

Comment:
Don't you have to add "i" as a suffix for integer types?  Likewise I think `tostring` is sufficient for decimal64, no?